framework/view/view_test.go

```diff
@@ -391,3 +391,43 @@ func TestSvelteImportFromNodeModule(t *testing.T) {
 	is.In(res.Body().String(), "<time datetime=\"2022-07-19 10:19:00\">Jul 19, 2022</time>")
 	is.NoErr(app.Close())
 }
+
+func TestSvelteImportFromSvelte(t *testing.T) {
+	is := is.New(t)
+	ctx := context.Background()
+	dir := t.TempDir()
+	td := testdir.New(dir)
+	td.NodeModules["svelte"] = versions.Svelte
+	td.NodeModules["svelte-time"] = "*"
+	td.NodeModules["livebud"] = "*"
+	td.Files["controller/controller.go"] = `
+		package controller
+		type Controller struct {}
+		func (c *Controller) Index() string { return "" }
+	`
+	td.Files["ui/Time.svelte"] = `
+		<h1>The Time</h1>
+	`
+	td.Files["view/index.svelte"] = `
+	 	<script>
+			import Time from "ui/Time.svelte"
```

Comment:
Oh, this is currently intentional. Paths that don't have `./` or `../` are considered node_modules. I'd definitely accept a change where we add support for `~/` to reference the project root, but for now, this is working as intended.

Reply:
Oh, ok, sorry for my misunderstanding of your last comment. However, I don't understand how you are meant to access files in the ui folder from a view. In the [documentation](https://denim-cub-301.notion.site/Hey-Bud-4d81622cc49942f9917c5033e5205c69#326e9e129af44fffaad0156f225ce3f2) it has the ui folder but doesn't show how to import from it:

$YOUR_APP
└─ ui
│  ├─ Calendar.svelte
│  └─ Popover.svelte
└─ view
   └─ users
      ├─ Avatar.svelte
      ├─ Card.svelte
      ├─ index.svelte
      └─ show.svelte

It then has a code snippet showing you how to import the Card.svelte but not the Calendar or Popover:

`<script>
  import Card from './Card.svelte'
  export let users = []
</script>

{#each users as user}
<Card {user} />
{/each}`

I'm sorry for the trouble and I hope I am not using bud in the wrong way.

Reply:
Hey @JanGordon , you'd be able to import it with the following:

```svelte
import Calendar from "../ui/Time.svelte"
```

Reply:
But I don't think that would work for any route. It would only work if the file is directly in the views folder and not nested in another folder, right? I'm a bit confused because I would have thought you would have to import components a lot. Am I trying to do things wrong? Or is there a way to reference the root of the project from anywhere?

Reply:
But I don't think that would work for any route. It would only work if the file is directly in the views folder and not nested in another folder, right? I'm a bit confused because I would have thought you would have to import components a lot. Am I trying to do things wrong? Or is there a way to reference the root of the project from anywhere?